examples/audit-report.html

```diff
@@ -0,0 +1,225 @@
+				<section class="well oag-rds">
+					<hgroup class="mb-3">
+						<h3 class="label label-primary fnt-hdng">News Release</h3>
+						<p class="fw-bold small text-uppercase"><time datetime="2025-03-18"><span>April 18, 2025</span></time></p>
+					</hgroup>
+					<p class="mb-3">2025 Spring Reporst of the Auditor General of Canada and the Commisioner of the Environment and Sustainable Development - Opening Statement.</p>
+				</section>
+			</div>
+		</div>
+		<div class="col-sm-8">
+			<h2>At a Glance</h2>
+			<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam viverra venenatis feugiat. Suspendisse eros est, consequat et sagittis ac, posuere aliquet mauris. Nunc eget quam sem. Suspendisse potenti. Nulla ornare erat ac tincidunt dapibus. Suspendisse at nunc eu lectus mollis viverra. Praesent efficitur massa ac lectus posuere, vitae pellentesque erat condimentum.</p>
+			<p>Interdum et malesuada fames ac ante ipsum primis in faucibus. Praesent tincidunt tortor fringilla, pretium lectus suscipit, placerat ex. Proin facilisis sapien erat, a molestie lorem consequat nec. Nulla sollicitudin pretium mi vitae commodo.</p>
+			<p>Nam at ex ac metus hendrerit condimentum eget et dolor. Quisque molestie libero et est pellentesque, vitae aliquet sapien posuere. Vestibulum at posuere risus, sit amet elementum nulla.</p>
+			<div class="row">
+				<div class="col-sm-6">
+					<h3>Why we did this Audit</h3>
+					<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p> 
+					<p>Nullam viverra venenatis feugiat. Suspendisse eros est, consequat et sagittis ac, posuere aliquet mauris. Nunc eget quam sem. Suspendisse potenti.</p> 
+					<p>Nulla ornare erat ac tincidunt dapibus. Suspendisse at nunc eu lectus mollis viverra.</p>
+					<p>Interdum et malesuada fames ac ante ipsum primis in faucibus. Praesent tincidunt tortor fringilla, pretium lectus suscipit, placerat ex. Proin facilisis sapien erat, a molestie lorem consequat nec. Nulla sollicitudin pretium mi vitae commodo. Proin vulputate pharetra dui at condimentum. Vestibulum vehicula nisi ut viverra facilisis.</p>
+				</div>
+				<div class="col-sm-6">
+					<h3>Highlights of our Recommendations</h3>
+					<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam viverra venenatis feugiat. Suspendisse eros est, consequat et sagittis ac.</p>
+					<p>Interdum et malesuada fames ac ante ipsum primis in faucibus. Praesent tincidunt tortor fringilla, pretium lectus suscipit, placerat ex. Proin facilisis sapien erat, a molestie lorem consequat nec.</p>
+					<h3>Key Factcs and Findings</h3>
+					<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam viverra venenatis feugiat. Suspendisse eros est, consequat et sagittis ac.</p>
+					<p>Interdum et malesuada fames ac ante ipsum primis in faucibus. Praesent tincidunt tortor fringilla, pretium lectus suscipit, placerat ex.</p>
+				</div>
+			</div>
+		</div>
+		
+		
+		<div class="col-xs-12 bg-light">
```

Comment:
This block shouldn't be inside the same "row" as the content above. Please move it out and remove the "col-xs-12" class.